### PR TITLE
WildFly 27.0.0.Final Compatibility Certification Request for Jakarta EE 10 Full Platform and Web Profile

### PR DESCRIPTION
**Pull request by scottmarlow**

### CODE REVIEW

File: WildFly_17.0.1.Final/jakarta-web-profile.adoc [hunk 1]
```diff
@@ -1,169 +0,0 @@
-= TCK Results
```
Comment (bstansberry) [inline WildFly_17.0.1.Final/jakarta-web-profile.adoc:1]: Was deleting this file intentional?

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc [hunk 2]
```diff
@@ -0,0 +1,557 @@
+= TCK Results
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc]: This folder should be WildFly_27.0.0.Alpha5

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+* Jakarta Batch
+* Jakarta Bean Validation
+* Jakarta Concurrency
+* Jakarta Contexts and Dependency Injection
+* Jakarta Debugging Support for Other Languages
+* Jakarta Dependency Injection
+* Jakarta Faces
+* Jakarta JSON Binding
+* Jakarta JSON Processing
+Jakarta Mail
```
Comment (bstansberry) [inline WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc]: Needs a '*'

File: WildFly_17.0.1.Final/jakarta-web-profile.adoc (hunk 1 above)
Comment (scottmarlow) [reply WildFly_17.0.1.Final/jakarta-web-profile.adoc:1]: Yes, I wanted to start a new EE 10 branch that doesn't include results from EE 8.  We probably could make that change on the `master branch` (change to `main` also).

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc (hunk 2 above)
Comment (scottmarlow) [reply WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc]: fixed

File: WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc (hunk 3 above)
Comment (scottmarlow) [reply WildFly_27.0.0.Final/jakarta-full-platform-jdk17.adoc]: fixed

File: WildFly_17.0.1.Final/jakarta-web-profile.adoc (hunk 1 above)
Comment (bstansberry) [reply WildFly_17.0.1.Final/jakarta-web-profile.adoc:1]: oh, sure, of course!

**Pull request merged**